frontend/lib/api/useApiGetRequest.ts

```diff
@@ -39,14 +47,13 @@ export function useApiGetRequest<T>(path: string): UseApiGetRequestReturn<T> {
           console.error("GET API error", e);
```

Comment:
Should there also be a setError here?

Reply:
My work in net ready for review yet ;-)

I was trying different approaches on this branch - but decided to split my changes in multiple PR's.

The flow of control / state in especially `PollingStationFormController` makes it a bit confusing on how and where to handle these errors. So first things first. I'll just introduce the error boundary in this PR to handle exceptions correctly and discuss refactoring some of the hooks (to improve readability / simplify control flow) next week.